lib/codecs/Cargo.toml

```diff
@@ -20,6 +20,7 @@ flate2.workspace = true
 influxdb-line-protocol = { version = "2", default-features = false }
 lookup = { package = "vector-lookup", path = "../vector-lookup", default-features = false, features = ["test"] }
 memchr = { version = "2", default-features = false }
+opentelemetry-proto = { path = "../opentelemetry-proto" }
```

Comment:
I think this should be optional and feature flagged, just like it is in `lib/vector-lib/Cargo.toml`